### ISSUE TITLE
As a special use administrator, I can accept applications that include Excel guide rosters.

### ISSUE THREAD

**Issue by lauraGgit** [18F/fs-permit-platform#223]
## Notes
The middle layer doesn’t accept Excel files for the guide roster. The form encourages guides to use an Excel template, but the middle layer API won’t accept it.

Severity: Severe - prevents application approval
Difficulty of remediation: Low - should just require adding an additional filetype to the schema

Mostly already enabled on the middlelayer. (just need to update test objects) see #111 .

For Platform need to update guide-documentation) toallowXls` here but need more digging: https://github.com/18F/fs-permit-platform/blob/master/frontend/src/app/application-forms/fields/file-upload.component.ts

## Acceptance Criteria
- [ ]

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] Server actions captured by logs
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by lauraGgit** [comment]
Moved #75 of the middlelayer to this repo, because it is more of a platform ticket.

**Comment by hursey013** [comment]
@lauraGgit Running locally I was able to successfully upload an `.xlsx` file as an applicant and confirmed that the file was also attached and available in the admin view. It appears that the logic to allow Excel specifically for "Guide identification" is in place as well: https://github.com/18F/fs-permit-platform/blob/8c0eae0ce417d5148f2600041106a27712724d6d/frontend/src/app/application-forms/temporary-outfitters/temporary-outfitters.component.html#L104
https://github.com/18F/fs-permit-platform/blob/8c0eae0ce417d5148f2600041106a27712724d6d/frontend/src/app/application-forms/fields/file-upload.component.ts#L97-L102

**Comment by lauraGgit** [comment]
ok, it looks like this is actually a middlelayer error - with the following error:

```
{
"level": "error",
"message": "ERROR: / POST Guide Documentation must be one of the following mime types: application/vnd.openxmlformats-officedocument.wordprocessingml.document, application/msword, text/rtf, application/pdf."
 }
```

**Comment by lauraGgit** [comment]
Mimetype added, this card is closed by [Middlelayer #120](https://github.com/18F/fs-middlelayer-api/pull/120)
